### PR TITLE
Adding get_field_display method for Value with choices

### PR DESCRIPTION
**Pull request by paulogiacomelli**
This PR purpose is to allow `Value` that have choices for getting the display value. The implementation is similar to Django `Field` get_FIELD_display.

Example (More on tests):
country = dbsettings.StringValue(choices=(("USA", "United States"), ("BR", "Brazil")))

`setting.country` would return USA
`setting.get_country_display()` would return United States

### CODE REVIEW

**Comment by zlorf** [comment]
Nice idea, thanks for preparing the pull request! I've left 2 small comments.

File: tests/models.py [hunk 1]
```diff
@@ -13,6 +13,7 @@ class TestSettings(dbsettings.Group):
     date = dbsettings.DateValue()
     time = dbsettings.TimeValue()
     datetime = dbsettings.DateTimeValue()
+    string_choices = dbsettings.StringValue(choices=(("String1", "String 1"), ("String2", "String 2")))
```
Comment (zlorf) [inline tests/models.py:16]: Can you make display values a bit more distinct from raw values? A single space is easy to miss.

File: dbsettings/values.py [hunk 2]
```diff
@@ -63,6 +64,15 @@ def contribute_to_class(self, cls, attribute_name):
 
         setattr(cls, self.attribute_name, self)
 
+        if self.choices is not None:
+            # Allows for adding get_FIELD_display() to fields with choices.
+            if 'get_%s_display' % attribute_name not in cls.__dict__:
+                setattr(
+                    cls,
+                    'get_%s_display' % attribute_name,
```
Comment (zlorf) [inline dbsettings/values.py:72]: Do not compute `'get_%s_display' % attribute_name` twice.

Comment (paulogiacomelli) [reply dbsettings/values.py]: No problem.

**Comment by paulogiacomelli** [comment]
Thanks for the review. I have updated the PR. Let me know if any other changes are needed.

**Comment by zlorf** [comment]
Cool, thanks!

**Pull request merged**